### DISCUSSION TITLE
机器人运行后 从qq群里@机器人发送/echo指令报错，Running Matcher failed。但是私聊机器人就没有问题

### DISCUSSION THREAD

**Discussion by njiubukedao** [nonebot/discussions#380] [问答] [answered]
机器人运行后 从qq群里@机器人发送/echo指令报错，Running Matcher failed。但是私聊机器人就没有问题

**Comment by njiubukedao** [comment]
nonebot cli version 1.1.2

**Comment by yanyongyu** [comment, marked as answer]
你的日志截图

**Reply by njiubukedao** [reply]
![image](https://github.com/nonebot/discussions/assets/95641450/53d9da74-f5df-43b1-b257-b911fa52a0cf)

**Reply by njiubukedao** [reply]
![image](https://github.com/nonebot/discussions/assets/95641450/f8cd1f32-f915-409e-9a5e-a7cdb74881dc)
go-cqhttp的日志

**Reply by yanyongyu** [reply]
这个报错与nb无关，是帐号被风控，请查看gocq日志

**Reply by njiubukedao** [reply]
好的谢谢
